### ISSUE TITLE
did you upload the lstm related code? and can you share the  experiment result?

### ISSUE THREAD

**Issue by chenren** [wadhwasahil/Video-Classification-2-Stream-CNN#3]

**Comment by stillbreeze** [comment]
We couldn't try out the LSTMs due to lack of enough GPU memory. If you wish to implement it, just follow the 2nd paper from the README. You need to compute the frame-wise feature vector from the FC layer of each video and feed it to a distinct timestep of an LSTM.

As for the results, I don't have a report, but the 2-stream CNNs yielded accuracy a bit lower than  those given in section 4.2 of the 2nd paper from the README.
Spatial - 65%-70%
Temporal - 50%-55%

**Comment by chenren** [comment]
the main consume comes from the feature extraction pre-frame, did
optical-flow helps save computation?

any idea about how to reduce consume in the front-end feature extraction
phase?


thks!

ren

2016-07-14 15:54 GMT+08:00 Ashar Javed <notifications@github.com>:

**Comment by guoyang007** [comment]
@stillbreeze I know we need to compute the frame-wise feature vector from the FC layer of each video and feed it to a distinct timestep of an LSTM,but how can we use  keras to make it ? the Sequential model can't work? I'm confused how to stack the outputs of the CNN layers to the LSTM,can you share some idea?

**Comment by kvignesh1420** [comment]
@wadhwasahil 

how many epochs did it take for the accuracy value to settle ? and how long did it take on your system. Please let me know the specifications.

cheers.

**Comment by wadhwasahil** [comment]
@kvignesh1420 In the code we used 50 epochs for the spatial stream, but I am pretty sure, we got the results before that. @stillbreeze Do you know the specs for temporal training?

**Comment by stillbreeze** [comment]
@guoyang007 : Sorry for the late reply. One straightforward way which most papers also follow is to not train the whole network end-to-end. So, you train the CNN, freeze the weights and then obtain the FC vectors and then feed them into a separate LSTM model.

@wadhwasahil : Sorry, but can't remember the training configuration.

**Comment by kvignesh1420** [comment]
@wadhwasahil 
@guoyang007 

can you please upload the lstm code if available. I am not able to extract the FC feature vectors and feed it into LSTM. Help me out.

**Comment by wadhwasahil** [comment]
Post some code so that we can help.

**Comment by kvignesh1420** [comment]
@wadhwasahil 
I have resolved the issue,

**Comment by wadhwasahil** [comment]
But still @kvignesh1420 , how did u solve it? It would be better for other people if you could share your views or code.

**Comment by kvignesh1420** [comment]
@wadhwasahil  give me some time. I will upload the whole procedure.. :D
